pcdswidgets/vacuum/base.py

```diff
@@ -270,6 +272,34 @@ def rotateIcon(self, rotate):
         if self.icon:
             self.icon.rotation = angle
 
+    @Property(str)
+    def ophydClass(self):
```

Comment:
```suggestion
    def expertOphydClass(self):
```
Maybe? It's OK as-is.

Reply:
I was aiming for a small name... It for sure looks better with the `expert` at the name...

Reply:
I will adopt this name as well as modify the class body variable to `EXPERT_OPHYD_CLASS` so it becomes a 1-1 mapping

Reply:
Addressed with https://github.com/pcdshub/pcdswidgets/pull/39/commits/44e502889dad60eafb881740eaec571897461802